notebooks/official/feature_store/vertex_ai_feature_store_feature_view_service_agents.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    # Wait for the LRO to finish and get the LRO result.
</pre><p>Would it be better to implement a check-result-and-wait loop here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2752/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p><code>lro.result()</code> will check and wait if needed. We could also use the GetOperation API and loop until the retrieved operation is done. I'm not sure if that's better. <code>lro.result()</code> seems more concise. What do you think? </p>

Reply:
<p>Oh Ok. It's a sync operation. Thanks.</p>